src/tokentype.js

```diff
@@ -95,7 +95,8 @@ export const types = {
   plusMin: new TokenType("+/-", {beforeExpr: true, binop: 9, prefix: true, startsExpr: true}),
   modulo: binop("%", 10),
   star: binop("*", 10),
-  slash: binop("/", 10)
+  slash: binop("/", 10),
+  starstar: binop("**", 11)
```

Comment:
Double-checking, I'm assuming the second number is the precedence and the higher the number, the higher the precedence? `**` has higher precedence than `*`.